server/src/server.ts

```diff
@@ -350,12 +350,22 @@ CallStack (from HasCallStack):
 		const charPos = errorJson.ce_position[1] - 1;
 		const suggestions = errorJson.ce_suggestions;
 		const actualMessage = errorJson.ce_errorMessage;
+		const offendingToken = errorJson.ce_offendingToken;
 
-		const start = { line: linePos, character:charPos };
+		const start ={ line: linePos, character: charPos };
+		const end = offendingToken ?
+				{ line: linePos, character: charPos + offendingToken.length }
+			:	{ line: linePos + 1, character: 0 };
```

Comment:
This is fine, but I noticed if Reach output doesn't have `offendingToken`, then highlighting to the end of the line would cause the entire line to be replaced by the quick fix that is chosen.  Once Reach is updated, will it always have the offendingToken (at least for cases where it has quick fix suggestions)?

Reply:
Yes, if `suggestions` is not empty, then there will be a non-null offending token. So, there shouldn't be a case where there's a quick fix that spans to the end of the line